readthedocs/projects/models.py

```diff
@@ -171,8 +172,8 @@ class Project(models.Model):
 
     # Other model data.
     path = models.CharField(_('Path'), max_length=255, editable=False,
```

Comment:
In some parts the function call is in this way 
```python
func(arg1, 
        arg2)
```

and other parts like this

```python
func(
    arg1,
    arg2)
```

Which style is the _correct_?

Reply:
The second one, but with a trailing comma and the ´)´ in the folowing line is closer to what we want. Like this

```
func(
    arg1,
    arg2,
)
```

but we use this only when it doesn't fit in 80 columns.

We have a style guide for this: http://docs.readthedocs.io/en/latest/contribute.html?highlight=contributing#contributing-to-development

We are using `pre-commit` command that runs a couple of scripts to apply the proper style to the code.